### ISSUE TITLE
Bear proposal: OclintBear

### ISSUE THREAD

**Issue by yash-nisar** [coala/coala-bears#1953]
> A static source code analysis tool to improve quality and reduce defects for C, C++ and Objective-C

See https://github.com/oclint/oclint for more information

bear proposal
area/lintbears
difficulty/low

**Comment by bhavishyagopesh** [comment]
@prachi1210 If you allow can I work on this?

**Comment by prachi1210** [comment]
@bhavishyagopesh Go ahead

**Comment by mridubhatnagar** [comment]
@bhavishyagopesh assign yourself to this issue.

**Comment by bhavishyagopesh** [comment]
@mridubhatnagar I don't have the access to do so, (i.e. `corobo assign <issue>` won't help)

**Comment by mridubhatnagar** [comment]
yes, corobo is down as of now. Later you can do it. Or someone who has the access will do it.
